README.md

```diff
@@ -51,29 +51,22 @@ before starting work on new features.
 
 1.  Install [yarn][yarn], the tool that manages AMY's JavaScript and CSS dependencies. [You can install it here][yarn].
 
-1.  Set up your local database with fake (development-ready) data with:
+1. Start running a local instance of Postgres and Redis. This requires Docker to be installed locally.  Redis is required to have certain features (like creating a new person and viewing a workshop request) work correctly.
 
     ~~~
-    $ make dev_database
+    $ docker-compose -f docker/docker-compose.yml -p amy up -d database redis
     ~~~
 
-1. Create cache tables for use with the database cache backend.
-
-    ~~~
-    $ python manage.py createcachetable
-    ~~~
-
-1.  Create an administrator account:
+1.  Set up your local database with fake (development-ready) data with:
 
     ~~~
-    $ python manage.py createsuperuser
```

Comment:
I totally understand if this command is here just to let folks who might be newer know it's available, but it's not needed as long as make dev_database is run